testdata/cmdtest/mutex/mutex.ct

```diff
@@ -1,10 +1,10 @@
-$ gromit mutex get testlock --etcduser ${TEST_ETCD_USER} --etcdpass ${TEST_ETCD_PASS} --script ${TEST_SCRIPT_PATH} --testlogs
+$ gromit mutex get testlock --host ${TEST_ETCD_HOST} --etcduser ${TEST_ETCD_USER} --etcdpass ${TEST_ETCD_PASS} --script ${TEST_SCRIPT_PATH} --testlogs
 {"level":"info","output":"Sleeping for 15 seconds\n","time":"0001-01-01T00:00:00Z","message":"script output"}
 
 # Runs with a non-existent script file - this should fail and the lock should be released immediately.
-$ gromit mutex get testlock --etcduser ${TEST_ETCD_USER} --etcdpass ${TEST_ETCD_PASS} --script ./non-existent-script.sh --testlogs --> FAIL
+$ gromit mutex get testlock --host ${TEST_ETCD_HOST} --etcduser ${TEST_ETCD_USER} --etcdpass ${TEST_ETCD_PASS} --script ./non-existent-script.sh --testlogs --> FAIL
 {"level":"fatal","error":"fork/exec ./non-existent-script.sh: no such file or directory","output":"","time":"0001-01-01T00:00:00Z","message":"could not execute script"}
 
 # Tests if the lock is released immediately from the above run.
-$ gromit mutex get testlock --etcduser ${TEST_ETCD_USER} --etcdpass ${TEST_ETCD_PASS} --script ${TEST_SCRIPT_PATH} --testlogs
+$ gromit mutex get testlock --host ${TEST_ETCD_HOST} --etcduser ${TEST_ETCD_USER} --etcdpass ${TEST_ETCD_PASS} --script ${TEST_SCRIPT_PATH} --testlogs
 {"level":"info","output":"Sleeping for 15 seconds\n","time":"0001-01-01T00:00:00Z","message":"script output"}
```

Comment:
aren't we missing the TLS ct?

Reply:
Yes, but the current remote etcd setup only have user auth. I'm trying to set up one with step-ca and step-cli. So doesn't make sense to add it before we have that.